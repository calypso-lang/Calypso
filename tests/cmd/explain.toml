bin.name = "calypso"
args = "explain E0001"
