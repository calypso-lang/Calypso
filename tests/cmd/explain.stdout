error: no extended information for error code: `E0001`
